src/configurations/destinations/http/ui-config.json

```diff
@@ -526,18 +526,21 @@
                     "to": "$."
```

Comment:
design was to show Destination event spec at LHS & Rudderstack source spec at RHS - https://rudderlabs.slack.com/archives/C075JU3BQTC/p1732860025165199?thread_ts=1732773966.221859&cid=C075JU3BQTC

Reply:
@sandeepdsvs can you re-check the design is being respected ? `Destination Property : Rudderstack Property` in all tabs. Adding screenshots in PR will be more helpful

Reply:
Do we need this in property mapping too? For headers and query parameters tabs it is destination -> rudderstack

Reply:
All the tabs should be consistent destination <- rudderstack - check out the design doc shared above

Reply:
Updated